jsonnet/config.libsonnet

```diff
@@ -0,0 +1,237 @@
+                action: 'replace',
+              },
+              {
+                sourceLabels: [
+                  '__meta_kubernetes_pod_label_openebs_io_storage_pool_claim',
+                  '__meta_kubernetes_pod_label_openebs_io_cstor_pool_cluster',
+                ],
+                separator: '',
+                targetLabel: 'storage_pool_claim',
+                action: 'replace',
+              },
+              {
+                sourceLabels: ['__meta_kubernetes_pod_label_openebs_io_cstor_pool', '__meta_kubernetes_pod_label_openebs_io_cstor_pool_instance'],
+                separator: '',
+                targetLabel: 'cstor_pool',
+                action: 'replace',
+              },
+              {
+                sourceLabels: ['__address__', '__meta_kubernetes_pod_annotation_prometheus_io_port'],
+                action: 'replace',
+                regex: '([^:]+)(?::\\d+)?;(\\d+)',
+                replacement: '${1}:${2}',
+                targetLabel: '__address__',
+              },
+            ],
+          },
+        },
+      },
+      Jiva: {
+        // To generate manifests for cstor. If set, manifests will be generated for Jiva.
+        enabled: true,
+        // To generate dashboards configMap yamls. If set, dashboards will be appended in grafana-dashboardDefinition yaml.
+        installDashboards: true,
+        // To generate prometheusRule yamls. If set, prometheusRule will be generated.
+        alertRules: true,
+        // ServiceMonitor configuration
+        serviceMonitor: serviceMonitor('metrics', '/metrics', {
+          'openebs.io/cas-type': 'jiva',
+        }) {
+          enabled: true,
+          endpoints+: {
+            // relabel configs to apply to samples before ingestion.
+            relabelings: [
+              {
+                sourceLabels: ['__meta_kubernetes_pod_label_monitoring'],
+                regex: 'volume_exporter_prometheus',
+                action: 'keep',
+              },
+              {
+                sourceLabels: ['__meta_kubernetes_pod_label_vsm'],
+                targetLabel: 'openebs_pv',
+                action: 'replace',
+              },
+              {
+                sourceLabels: ['__meta_kubernetes_pod_label_openebs_io_persistent_volume'],
+                targetLabel: 'openebs_pv',
+                action: 'replace',
+              },
+              {
+                sourceLabels: ['__meta_kubernetes_pod_label_openebs_io_persistent_volume_claim'],
+                targetLabel: 'openebs_pvc',
+                action: 'replace',
+              },
+              {
+                sourceLabels: ['__meta_kubernetes_pod_label_app'],
+                targetLabel: 'openebs_jiva_label',
+                action: 'replace',
+              },
+            ],
+          },
+        },
+        // PodMonitor configuration
+        podMonitor: {
+          enabled: false,
+        },
+      },
+      lvmLocalPV: {
```

Comment:
Can we add one for device localpv as well?

Reply:
Yes, will add it.